docs/application/dotnet/guides/user-interface/nui/vectorgraphics/CanvasView.md

```diff
@@ -0,0 +1,40 @@
+# CanvasView
+
+A CanvasView is a NUI control that displays vector primitives.
```

Comment:
@mijin85cho, you're absolutely right about this. Typically any word starting with the letter "n" is preceded by the article "a" but in this case, since the letters in NUI is pronounced individually and the sound starts with "en.." it should be preceded by the article "an".
Thank you for pointing it out.

Reply:
Modified